src/icons/serverless-deploy-step.svg

```diff
@@ -0,0 +1 @@
hWcjb2EN4B9kjsAJwMlaV/qN3Qq3eQHIPbwbebfJnz9M7sjO3ADrApnNdHZnDvWw0o6fAPI6fmSPLvpzh4QCrANkPl3/6uF1LLTvK4Dc5lNWWpUDPLxDWAROAE7WqoYWJtWcaJNY2iZAZrcEmlfXAH5aGsAqQWY1J1iZgDItpQY5asySf+1dcoflA6wKZGeA1Zw4ViaGua2lBDmqjZN39aD06JvA276WObRgb1xiJbPKuJQK5Kg2qg3gBGZZILP33gVLp8vIcHkiMgd49i0tETgBOFnLAJm9PpeVjy3zYnVEZnlgb+YNIwCWBTKb7xc6Y2Vm2PKIHHnkTrxsFMQMZpER2Tm5jcLaVOkhZgJYG5HDlVPGQSwSZGeITdcvRs0JFX9p1oLMnw9em16U/LduwlpERK6deozYq3Wx3FPAWpDD5Yq1Edkd/809D2KLK2AtyBQ2+KxlE6Jw+zmIiMhsNnX93Hcocm8B4zUF7AWZ33q+Ru74PqMisyiQGcws7Rbc+gdgtvlir+XdsEH+9feNgVkkyEmdCz7/rsA1KVq+yrFhdURu1yVcOZdqcmj7MEDGtgyQeXQe2k7hyqV2k0u1XRqQmVf5DZLpX2uNzrJA5jAPbiJ36p1SAZwYWyqQudFREM98lvjMca9ILhPkZKjBXhQZ+XcSH5diXT6Q19wa1Sa0XAiqAJlHZ3YhWKJnkksLMuc5rJN//Q9KhxqqQE6ic/PKi0SRb31ULjfIa+4Nl0+SW/2hEqBVg8yArp/+BoWro1bDDJDX3BtfCP5KOsw6QOZDDX4h+K61MAPkdte2LgTlTX3SBXIy1OAXgt5yu9VWbAPkddwYOsPSbm/rBplH58pWChY+Xcfy4n4FkLv5LliVMi3KBJCT6BxfCAbdFCjU9wB5A3fFF4LPChs7mwQyA7p+5vFCvBByAzfZ+2D9RoZn+T1y58ibelUIzKaBzIcag5v5iyqzaGLavojIaT0S+XEp2Zw1MEwEORlqxBeCK2kVMWo/gJzRHeHqJXLH7n83SK/b0u2/mQwyj85sHmABHz4CyBlBZrtH/gp5s2/3NdQwHWQOc2VL4Z7VAMh9gBwfEsWvHhvJVjO5CCAzmN3xN/tWRseBADmn6qwAoje5P3V0LgrIjUu7cyqj9nCALEDvYOk/1oHcrP5CgDLqmgDIObUO7vw3UzXPIkRkVgyRFTsv0gKQc3grWDyWCWKWvTAdZD6hdeGTHKroORQg96k7f8NpHwVgTAWZFQfnt6yD1T4V0XsYQO5D/34hNjUiNy7soKg+04cS5hwCkDP6Irj1YeoLu/YbIcm2SRG5fu5Ja8rRAuTUIEfk3/xrLohNici1U1+j4KZdhV0AciqQI/Jv/Dk3xLpBdk48Qu7Me6ksLtpOAHlDj4Xkz/1JCMS6QHaOPxTfqbN4EipA7glyKLzclsoxMstEsHJaZahxAZC7gRwF5F/7vbBIrPpij2Uiwtp0N+us+x4gr+fSyJcyzUnF0CLORJxfzyqrvwPIne4NXfJmfys8EsuOyM4Qy0R83GlNaT4D5HZXh03yZt6UBrGMiOxUtlqbiWh3zUbbADlRKKyTN12cAi1OZS0TEbqJBaVeA2Tm/qBG3tQvpUZiUUOLVibCWyo1uJ3GA+TgLnmTP1cCcd6hRdkyEZ2w9vpcapAjf5m8yVeUQdwvyPWzT1C4Ur5MRC9wO38rLciRt6jlzahZbog4Q9tLnYnohLXX51KCzCpvuuMvKY3EWcbIyET0Qnb930oHcuTOSytQmMDaa90rIjuDm6lZfY0ImYj1ae3xbalAZqWv3LEXtETiBO71QHYGHoyfiUAmogeqvX8qDchRY5bc0T1aIWYwd4IcZyKmensJv26oQClAjhrTRkDcDjLPRCx/saGDsEM6BawHOaqNkVvdrT0SJ0OL+ufftm52RjrU5O5lNcihU1X2kpsE1K7riZ8Re/sqUSTXoyVt3VqQedXMkV36I/HoHl4jjiI7KsOb+ndiJcjh3YvkZiwu2DWS9lG7grdVfZbXU6awYarvrTov60DmL08feUZfJB7ZSf78YWK3v7GoU8AqkMOVM+SO7NQGMXsgP2peU+c99NRSwBqQgzuDmeuwiRpOeNMHiF1YYtGngBUgB3cG9ECMTIQ+cjt6LjzIweI/1UPMMhELRwmZiA6aNH4sNMh5ign2NaxAJkIjqr27LizIwe2P1F3UIRPRmyIDfi0kyHkrYmaJxshEGEBpilMoGMismGD+iphpQGaTUZGJSEGQIbsUCGRxFTF7gjzxMp6JMATOLKdREJBZRcw/yh0TIxORhRvj9i0AyOIrYv5fRK7uwjMRxmGZ/YTMBplXxHxHUiR+mvy5Q3gmIjszRh5hLsgSK2IiE2Eki7lOykyQJVXERCYiFytGH2weyGFDfEVMZCKMhlDEyZkFMismKLIiZisT4YvQCm0YrIA5IIusiNnKRNQNlh6nJlIBM0AWVhETmQiRcBSpLe0gi6qIGWcirhZJe5yrQAW0giyiImaciRgRKAmaKqIC2kCOK2Lu6/9mx/hPKFw+iToRRaROwjlrAZlD3G8xwdG9FCx9gtkZEmAocpNaQPamX88eiXkm4kNCnYgi4ybv3JWDzKNxpqInO8if/wDPRMhjwIqW1YPMyrumBBmZCCsYU2KEcpDZ0MAdfa4nzHgmQonvrepEPcjstXaLx9YHefynxKoFoWKlVYwpMUYLyMyycPk4eZP7ebFB9oowZCKU+NvaTrSBbK2iMEyLAgBZi+zoVLQCAFm0omhPiwIAWYvs6FS0AgBZtKJoT4sCAFmL7OhUtAIAWbSiaE+LAgBZi+zoVLQCAFm0omhPiwIAWYvs6FS0AgBZtKJoT4sCAFmL7OhUtAIAWbSiaE+LAgBZi+zoVLQCAFm0omhPiwIAWYvs6FS0AgBZtKJoT4sCAFmL7OhUtAIAWbSiaE+LAgBZi+zoVLQC/wM5KqqK2uLdDgAAAABJRU5ErkJggg=="/></defs></svg>
```

Comment:
These are general change we make with svgs.
remove fill="none" in svg tag
change fill="currentColor" in path tag
Please make the change unless otherwise required

Reply:
There are multiple colors in this one, so can't do it.